src/tex/ms.tex

```diff
@@ -287,6 +287,13 @@ \subsection{Curation workflow}\label{sec:curation}
 \paragraph{Semi-programmatically generated questions}
 In addition to the manually curated questions, we also generated questions programmatically. An overview of the sources of the semi-programmatically generated questions is provided in \Cref{tab:semi_programatically_sources}.
 
+\subparagraph{Chemical Preference data}
+
+These questions assess the ability to establish a \enquote{preference}, such as favoring a specific molecule. Chemical preference is of major importance in drug discovery projects, where the optimization process to reach the desired molecular properties is a process that lasts several years within a chemist's career.
+Our data corpus is adapted from the published dataset by \textcite{Choung2023}, which consists of more than 5000 question-answer pairs about chemical intuition. To build the dataset, they presented 35 medicinal chemists with two different molecules, asking them what molecule they would like to continue with when imaging an early virtual screening campaign setting. The question was designed so the scientists do not spend much time answering it, relying only on their feelings or \enquote{chemical preference}.
+
+To understand whether the capabilities of the leading models align with the preferences of professional chemists, we selected 1000 data points from the original dataset to create a meaningful evaluation set, where molecules are represented as SMILES.
+We only considered molecules for which we could obtain IUAPC names to ablate the effect of different molecular representations. 
```

Comment:
```suggestion
These questions assess the ability to establish a \enquote{preference}, such as favoring a specific molecule. Chemical preference is of major importance in drug discovery projects, where the optimization process to reach the desired molecular properties is a process that lasts several years within a chemist's career.
Our data corpus is adapted from the published dataset by \textcite{Choung2023}, which consists of more than 5000 question-answer pairs about chemical intuition. To build the dataset, they presented 35 medicinal chemists with two different molecules, asking them what molecule they would like to continue with when imaging an early virtual screening campaign setting. The question was designed so the scientists do not spend much time answering it, relying only on their feelings or \enquote{chemical preference}.

To understand whether the capabilities of the leading models align with the preferences of professional chemists, we selected 1000 data points from the original dataset to create a meaningful evaluation set, where molecules are represented as SMILES.
We only considered molecules for which we could obtain IUPAC names to ablate the effect of different molecular representations. 
```

perhaps it would be good to also state how the selection was done: randomly, maximally diverse using Tanimoto similarity between Morgan fingerprints, ...?